File: src/main/java/eterea/core/service/model/dto/StockAndArticulosDto.java
package eterea.core.service.model.dto;

import eterea.core.service.hexagonal.stock.articulomovimiento.domain.model.ArticuloMovimiento;
import eterea.core.service.hexagonal.stock.stockmovimiento.domain.model.StockMovimiento;
import lombok.*;

import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class StockAndArticulosDto {

    private StockMovimiento stockMovimiento;
    private List<ArticuloMovimiento> articuloMovimientos;

}

File: src/main/java/eterea/core/service/kotlin/model/dto/StockAndArticulosDto.kt
package eterea.core.service.kotlin.model.dto

import eterea.core.service.kotlin.model.ArticuloMovimiento
import eterea.core.service.kotlin.model.StockMovimiento

data class StockAndArticulosDto(

    var stockMovimiento: StockMovimiento? = null,
    var articuloMovimientos: List<ArticuloMovimiento>? = null

)
